packages/repository-tests/src/crud/relations/acceptance/has-many-inclusion-resolver.relation.acceptance.ts

```diff
@@ -187,9 +187,92 @@ export function hasManyInclusionResolverAcceptance(
       expect(toJSON(result)).to.deepEqual(toJSON(expected));
     });
 
-    it('throws when navigational properties are present when updating model instance', async () => {
-      const created = await customerRepo.create({name: 'customer'});
-      const customerId = created.id;
+    skipIf(
+      !features.hasRevisionToken,
+      it,
+      'returns inclusions after running save() operation',
```

Comment:
Could you please provide more details on why are these new test cases skipped? IIUC, you are running these tests only for CouchDB/Cloudant and skipping them for all other databases (memory, PostgreSQL, MongoDB). The code in the test looks legit to me, I find it suspicious that it does not work on so many database types.

Ideally, I'd like to find a way how to enable these tests for all databases we support.

If that's not possible or if it requires too much effort, then I would like you to add code comments near `!features.hasRevisionToken` line to explain why each test is skipped.

Reply:
The test should run on all databases except `Cloudant` cause it needs the `_rev` property. I will add comments to clarify.

Reply:
@agnes512 thank you for replying in time, I am running these 2 tests and I think I understand why they fail for Cloudant and mongodb, fixing it.

Reply:
@bajtos The test case here skips Cloudant connector because the [Customer model](https://github.com/strongloop/loopback-next/blob/master/packages/repository-tests/src/crud/relations/fixtures/models/customer.model.ts#L23) in fixtures doesn't have `_rev` property defined. 

`skipIf(features.hasRevisionToken, testcase){//tests}`means when `features.hasRevisionToken` is true(which only applies for the cloudant connector), the test will be skipped.

I am not very familiar with the shared test case, so agreed with Agnes's solution, 
if you want to enable same tests for cloudant, I can add them in a separate PR (within same story).

Reply:
@bajtos The test fails for mongodb because the id type is `object` not string, therefore fails at the check in https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/dao.js#L2621, hmm...any suggestion on how to fix it?